cli/src/cmd/version/version

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
```suggestion
2.0.0
```

Reply:
The first version of the CLI was 0.9.1, so this would then be the first major release

Reply:
I understand your thinking, but I don't think we should continue to mislabel our version here. The previous CLI was version 1 and was released as a major release, the version number was just incorrect. It's just going to cause confusion everywhere. Cause we'll be saying it's version 1.1.3 or whatever when it's actually version 2.1.3. This is a completely new CLI and as such, I think it's important we accurately tag it instead of continuing with the incorrect tagging of the past.